modules/nf-core/bowtie2/align/tests/main.nf.test

```diff
@@ -44,44 +69,37 @@ nextflow_process {
             assertAll (
                 { assert process.success },
                 { assert snapshot(
-                    file(process.out.bam[0][1]).name,
+                    process.out.bam.collect { meta, bamfile -> bam(bamfile).getReadsMD5() },
```

Comment:
Since sort is off but
```suggestion
                    // NOTE: Since sort is off reads aren't going to be deterministic
                    process.out.bam.collect { meta, bamfile -> bam(bamfile).getHeaderMD5() },
```